inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/ClusterConfigServiceImpl.java

```diff
@@ -46,28 +56,61 @@ public class ClusterConfigServiceImpl implements ClusterConfigService {
     @Autowired
     private ClusterConfigEntityMapper clusterConfigEntityMapper;
     @Autowired
-    private InlongClusterService clusterService;
+    private InlongClusterEntityMapper clusterEntityMapper;
+    @Autowired
+    private InlongClusterOperatorFactory clusterOperatorFactory;
 
     @Override
     public boolean refresh(String clusterTag, String operator) {
         try {
             ObjectMapper objectMapper = new ObjectMapper();
-            List<ClusterInfo> clusterInfos = clusterService.listByTagAndType(clusterTag, ClusterType.PULSAR);
-            List<PulsarClusterConfig> list = new ArrayList<>();
-            ClusterConfigEntity existEntity = clusterConfigEntityMapper.selectByClusterTag(clusterTag);
-            for (ClusterInfo clusterInfo : clusterInfos) {
-                PulsarClusterInfo pulsarCluster = (PulsarClusterInfo) clusterInfo;
-                PulsarClusterConfig pulsarClusterConfig = CommonBeanUtils.copyProperties(pulsarCluster,
-                        PulsarClusterConfig::new);
-                pulsarClusterConfig.setVersion(pulsarCluster.getVersion());
-                pulsarClusterConfig.setClusterName(pulsarCluster.getName());
-                pulsarClusterConfig.setServiceUrl(pulsarCluster.getUrl());
-                list.add(pulsarClusterConfig);
+            ClusterPageRequest request = ClusterPageRequest.builder()
+                    .clusterTag(clusterTag)
+                    .typeList(Arrays.asList(ClusterType.TUBEMQ, ClusterType.PULSAR, ClusterType.KAFKA))
+                    .build();
+            List<InlongClusterEntity> clusterEntityList = clusterEntityMapper.selectByCondition(request);
```

Comment:
is it possible that clusterEntityList is null?

Reply:
Fixed.